mlir-sycl/include/mlir/Conversion/SYCLToLLVM/SYCLToLLVMPass.h

```diff
@@ -20,6 +20,9 @@ namespace mlir {
 class ModuleOp;
 template <typename T> class OperationPass;
 
+#define GEN_PASS_DECL_CONVERTSYCLTOLLVM
+#include "mlir/Conversion/SYCLPasses.h.inc"
```

Comment:
Convention is to use per-class declaration in conversion passes and single `GEN_PASS_DECL` in transform passes.

Reply:
```suggestion
#define GEN_PASS_DECL_CONVERTSYCLTOLLVM
#include "mlir/Conversion/SYCLPasses.h.inc"
#undef GEN_PASS_DECL_CONVERTSYCLTOLLVM
```